### ISSUE TITLE
Speed up Conversation opening

### ISSUE THREAD

**Issue by yvolk** [andstatus/andstatus#361]
Currently messages, which are parts of one conversation, are loaded one-by-one, this is why the process is too long even when most of messages are already present in a local database cache.

We need to create our local conversation id and link all related messages by this ID right during message storage to database. This will increase saving time a bit (this is done in a background), but greatly improve Conversation View opening.

This is partially related to #328 "Support Conversation API", but actually is a separate task, which should be done before implementation of #328.

**Comment by yvolk** [comment]
Implemented in v.31
